certificate-policy.md

```diff
@@ -888,6 +902,8 @@ The CA shall take appropriate administrative and disciplinary actions against pe
 #### 5.3.7 Independent Contractor Controls
 Direct contractor personnel employed to operate any part of the CAs or perform functions pertaining to the infrastructure shall be subject to the same personnel requirements set forth in this CP.
 
+The FPKIPA shall be notified when a contractor operating a CA under this policy transfers to a different organization by acquisition, contract, or other means. The CA owner, in coordination with the FPKIPA, shall ensure the new organization can meet the requirements of this CP.
```

Comment:
I don't think this is necessary?  Can you provide an example for when this would occur AND meet the definitions outlined in the mozilla policy?

Reply:
Mozilla states: "This section applies when operation of a certificate in Mozilla's root program is transferred to a different organization, whether by acquisition or contract. The transferor MUST ensure that the transferee is able to fully comply with this policy. The transferor will continue to be responsible for the root certificate's private key until Mozilla has been provided with an audit statement (or opinion letter) confirming successful transfer of the root certificate and key. Issuance MUST NOT occur until the transferee has provided all the information required by the CCADB, and demonstrated to Mozilla that they have all the appropriate audits, CP/CPS documents and other systems in place."

Mozilla policy divides CA ownership (8.1) and CA operation (8.2).  This change is in response to 8.2. The government follows a government owned, contractor operated model so the operator (the contractor) should be familiar with the CP (and Mozilla) requirements. If operation is government then this would never apply.